federatedscope/vertical/caesar_v_fl/README.md

```diff
@@ -0,0 +1,18 @@
+### Caesar Vertical Federated Learning
```

Comment:
Maybe we can move this README.md to `/federatedscope/vertical/` and merge it with that of secure_LR

Reply:
ok